content/specific-skill-success-criteria/basic-data-analysis/_index.md

```diff
@@ -5,4 +5,151 @@ title: 'Assessment: Basic data analysis'
 topic_needs_review: true
 ---
 
-TODO
\ No newline at end of file
+All students must understand all of the following concepts:
+
+- Statistical concepts:  Mean, Median and Mode
+- Statistical concept:  IQR (Interquartile range)
+- Statistical concepts:  Standard deviation and Variance
+- Statistical concepts:  Correlation and p-value (Probability value) and Null-hypothesis vs. Alternative-hypothesis
+- Statistical concepts:  Standard deviation and Standard error
+
+### Statistical concepts: Mean, Median and Mode
+
+#### Mean (Average value):
+```
+Calculating the mean (average value):
+
+list_of_speed_data = [87, 94, 78, 77, 85, 86]
+
+mean = (87+94+78+77+85+86) = 507/6 = 84.5
+```
+
+
+#### Median (Middle value):
+```
+Calculating the median (middle value):
+
+list_of_speed_data = [100, 200, 300, 400, 500, 600]
+
+When we have an even amount of numbers we need to calculate the mean (average) of the two middle 
```

Comment:
This shouldn't be giving lengthly explanations. These are assessment criteria.

This criterion is basically:
The student must be able to calculate the median value when given a list of values. They should be able to deal with lists that have an even number of elements. They should also be able to deal with lists that have an odd number of elements.

We can check this by giving the student a list of numbers and asking them "what is the median?"